src/helperFunctions/beautifyTaskStatus.ts

```diff
@@ -0,0 +1,19 @@
+import task from '../interfaces/task.type';
+
+const beautifyTaskStatus = (tasks: Array<task>) => {
+  const taskList: Array<task> = [];
+  tasks.forEach((taskItem: task, taskIndex) => {
+    taskList.push(taskItem);
+    const splitedStringArray = taskItem.status.split('_');
+    splitedStringArray.forEach((subString, index) => {
+      splitedStringArray[index] = subString.toLowerCase();
```

Comment:
Better to make status as lowercase first and then use split.

Reply:
Thank you 
Will update it 👍